app/src/main/res/layout/activity_main.xml

```diff
@@ -2,35 +2,36 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<merge
-        xmlns:android="http://schemas.android.com/apk/res/android"
-        android:layout_height="match_parent"
-        android:layout_width="match_parent">
+<FrameLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/rootView"
+    android:layout_height="match_parent"
+    android:layout_width="match_parent">
 
     <FrameLayout
-            android:orientation="vertical"
-            android:id="@+id/container_web_render"
-            android:layout_width="match_parent"
-            android:layout_height="match_parent"/>
+        android:orientation="vertical"
+        android:id="@+id/container_web_render"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"/>
```

Comment:
Looks like there were some unintended indentation changes in this file.

Reply:
It was improperly indented before actually, this was intended